src/ORM/AssociationCollection.php

```diff
@@ -29,6 +30,7 @@ class AssociationCollection implements IteratorAggregate
 {
 
     use AssociationsNormalizerTrait;
+    use LocatorAwareTrait;
```

Comment:
I imagined that you would have added the locator as an argument to the constructor and pass it there when building the object.

It is ok to have this trait, but better to have it in the constructor as well.

Reply:
Good idea :+1: